.editorconfig

```
@@ -6,9 +6,11 @@ end_of_line = lf
 charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
+tab_width = 4
 
 [*.json]
 indent_style = space
+indent_size = 4
```

For whatever reason, it seems to always be 2 (ex npm install will use 2), so let's not fight something we'll rarely look at

```suggestion
indent_size = 2
```